charm/docs/explanation/charm-architecture.md

```diff
@@ -0,0 +1,30 @@
+# NetBox charm architecture
+
+The NetBox charm initial version has been generated with the 
+help of the 12 Factor Project. The 12 Factor Project provides many 
```

Comment:
Per https://docs.google.com/document/d/1uqm1S_ZRVCWwklHerfrIE-yrZb7bLLzjYsaepMM1pTQ/edit this is now named the paas-app-charmer project. We should check exactly how to refer to it in this context.